client-java/controller-api/src/main/java/org/evomaster/client/java/controller/api/dto/database/schema/CompositeTypeColumnDto.java

```diff
@@ -0,0 +1,40 @@
+package org.evomaster.client.java.controller.api.dto.database.schema;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CompositeTypeColumnDto {
+
+    /**
+     * The name of the composite type
+     */
+    public String name;
+
+    /**
+     * Returns if the column is a composite or a built-in type
+     */
+    public boolean isCompositeType;
```

Comment:
bit confusing... so a `CompositeTypeColumnDto` might have `isCompositeType` false???

Reply:
you might have something like this
CREATE TYPE myDomain AS (
  integerColumn integer NOT NULL
);
In this case, "integerColumn" is indeed a composite column but "integer" is not a composite type.

In contrast,
CREATE TYPE nestedType AS (
  myDomainColumn MyDomain NOT NULL
);
In this case, "myDomainColumn" is a composite type column but type "MyDomain", therefore isCompositeType==true. 

I agree the field name is confusing, instead of "isCompositeType" we could use "columnHasNoBuiltInType" ?

Reply:
yep, i guess that could be an option. can also add that example as a comment in the code to clarify it

Reply:
done!